src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/BlendShapeIndex.cs

```diff
@@ -51,13 +51,13 @@ namespace Tizen.NUI.Scene3D
     /// BlendShapeIndex blendShapeIndex1 = new BlendShapeIndex(new PropertyKey("node1"), new PropertyKey("Smile"));
```

Comment:
please shorten the `<summary>`, maybe separate 2 `<example> description <code>... </code></example>` sections

Reply:
Fix done.